src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import './styles.scss'
+import Pic1 from '../../images/2.png'
+import Pic2 from '../../images/4.png'
+
+import Button from '../../commonComponents/Button/Button.js'
+
+function HomePage() {
+    return (
+        <main>
```

Comment:
this main may go to App.js because it will have same styling on all the pages. use fragment instead